tests/unit-tests/dds/DCPS/WaitSet.cpp

```diff
@@ -7,8 +7,11 @@
 
 #include <gtest/gtest.h>
 
+#include <dds/DCPS/Service_Participant.h>
```

Comment:
As this test creates particpants, waits, etc., move it to `/tests/DCPS` and add it to the `*.lst` files as appropriate.

Reply:
Test moved to `/tests/DCPS/WaitSetLivelinessLost`